src/services/WolfDen.Domain/Entity/LeaveDay.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace WolfDen.Domain.Entity
+{
+    public class LeaveDay
```

Comment:
Change table name to LeaveRequestDay

Reply:
Name Changed as Requested
LeaveDay to LeaveRequestDay